intake/catalog/tests/conftest.py

```diff
@@ -23,7 +23,7 @@ def ping_server(url, swallow_exception):
         else:
             raise e
 
-    return r.status_code == 200
+    return r.status_code in (200, 403)  # allow forbidden as well
```

Comment:
I thought it would still be possible to "see" the server even when we have no permissions to access any of the data - it looks like an empty cat. Is this wrong?

Reply:
There are two checks done by the auth plugins: one to determine if the connection is allowed, and one to determine if the data source is available to the client.  If the first check is denied, automatic 403 on every request.  If the second one fails, then the catalog listing will be empty, and attempting to open the source will return the 403